tooling/create-tauri-app/src/templates/dominator/_Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "<%= name %>"
+version = "1.0.0"
+categories = ["wasm"]
+edition = "2018"
+
+[profile.release]
+lto = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[workspace]
```

Comment:
The build fails because cargo thinks the projected generated is in a workspace because of this line, is it necessary ?

Reply:
It's not technically necessary, but it's *very* useful for multi-crate packages (which are common on the web). It shouldn't cause any harm. What error are you getting?

Reply:
```
error: current package believes it's in a workspace when it's not:
current:   D:\Development\Projects\tauri-app\src-tauri\Cargo.toml
workspace: D:\Development\Projects\tauri-app\Cargo.toml

this may be fixable by ensuring that this crate is depended on by the workspace root: D:\Development\Projects\tauri-app\Cargo.toml
Alternatively, to keep it out of the workspace, add the package to the `workspace.exclude` array, or add an empty `[workspace]` table to the package's manifest.
Error: failed to build app

```

Reply:
Where is `src-tauri` coming from? That isn't included in this recipe.

Reply:
`src-tauri` is added when `create-tauri-app` runs `tauri init`, this folder contains the logic of the rust-backend of the app

Reply:
Ahhh, interesting, that explains it. So the problem is that `src-tauri` is a sub-crate, but it isn't included in the workspace.

The easiest way to fix it is to just add it to the workspace in `Cargo.toml`:

```toml
[workspace]
members = [
    "src-tauri",
]
```

However, this will cause the backend to be a sub-crate of the frontend.

Personally I prefer having 3+ crates:

* A root crate which is a workspace.
* A backend crate (which I usually call `backend` or `server`)
* Multiple frontend crates (e.g. `foo`, `bar`, etc.)

So perhaps the best solution is to move the frontend code into a `src-frontend` crate. But that seems like something which should be standardized for all the recipes, not just dominator.

Reply:
I think adding `src-tauri` to the workspace is fine for now.